src/index.css

```diff
@@ -21,6 +21,12 @@ body {
   height: 100vh;
 }
 
+.splash-container > .splash {
+  border: 1px solid #000;
+  border-radius: 2rem;
+  box-shadow: 8px 14px 8px grey;
+}
```

Comment:
Hey Brother @Cosmosha I see, you did a great job by changing your code suggested by Moyasi. Thanks so much for that.
I've noticed one thing you put the styles for the splash in the index.css file which is good.


What if we could create a new styles folder and then create a file called splash.css inside it?
Then we can find the styles related to splash so easily. right? Let me know what you think.